app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/messages.js

```diff
@@ -81,4 +81,8 @@ export const messages = defineMessages({
     id: 'IntegrationsDescription.integrationList',
     defaultMessage: 'Integration List',
   },
+  linkToDocumentation: {
+    id: 'IntegrationsDescription.integrationHeadDocumentationLink',
```

Comment:
Please use the same naming as for key in messages object: `IntegrationsDescription.linkToDocumentation`.